app/controllers/report_controller/reports/editor.rb

```diff
@@ -169,7 +169,7 @@ def build_edit_screen
     build_tabs
 
     get_time_profiles # Get time profiles list (global and user specific)
-    cb_entities_by_provider if Chargeback.db_is_chargeback?(@edit[:new][:model])
+    cb_entities_by_provider if Chargeback.db_is_chargeback?(@edit[:new][:model]) && [ChargebackContainerImage, ChargebackContainerProject].include?(@edit[:new][:model].safe_constantize)
```

Comment:
@zeari method cb_entities_by_provider is only for `ChargebackContainerImage, ChargebackContainerProject`, right ?